src/scss/components/_scenarios.scss

```diff
@@ -0,0 +1,103 @@
+.scenarios {
+  $main: &;
+
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 230px;
+  padding-top: 30px;
+  position: relative;
+  width: 100%;
+
+  &__title {
+    font-family: $hkgrotesk-bold;
+    font-size: $mobile-title;
+    height: 54px;
+    line-height: 30px;
+    margin-bottom: 102px;
+    margin-left: auto;
+    margin-right: auto;
+    text-align: center;
+    width: 227px;
+  }
+
+  &__principal-container {
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: center;
+    margin: auto;
+    padding-left: 100px;
+    padding-right: 100px;
+    position: relative;
+  }
+
+  &__container-images {
+    display: flex;
+    flex-direction: column;
+    position: relative;
+  }
+
+  &__text {
+    font-family: $hkgrotesk-bold;
+    font-size: 18px;
+    margin-bottom: 20px;
+    margin-top: 20px;
+  }
+
+  &__images--0 {
+    display: flex;
+    height: auto;
+    position: absolute;
+    right: 60px;
+    text-align: right;
+    top: -70px;
+    width: 33%;
+    z-index: 1;
+  }
+
+  &__images--1 {
+    display: flex;
+    flex-direction: column;
+    height: auto;
+    position: relative;
+    width: 100%;
+    &::before {
+      background-image: url('../static/assets/FormaAzul.png');
```

Comment:
same here, the name of this image file, it should not be capitalized, and the words separated with (-)

Reply:
Done. I changed the name of the images with the convention they should have.